MoreLinq.Test/EndsWithTest.cs

```diff
@@ -92,11 +92,12 @@ public void EndsWithUsesSpecifiedEqualityComparerOrDefault()
             Assert.True(first.EndsWith(second, EqualityComparer.Create<int>(delegate { return true; })));
         }
 
-        [Test]
-        public void EndsWithUsesCollectionsCountToAvoidUnnecessaryIteration()
+        [TestCase(true)]
+        [TestCase(false)]
+        public void EndsWithUsesCollectionsCountToAvoidUnnecessaryIteration(bool readOnly)
         {
-            var first = new BreakingCollection<int>(1, 2);
-            var second = new BreakingCollection<int>(1, 2, 3);
+            var first = new[] { 1, 2 }.ToBreakingList(readOnly);
```

Comment:
Use `ToBreakingCollection`, like you did in 1499147c92efbf92ef7c804bf4dbcc6756894448 for PR #448.

Reply:
@atifaziz this fix was already done.